### PR TITLE
test: Increase [integration-tx-api] module test case coverage

### LINKED ISSUE #6507: task: Improve the test case coverage of [integration-tx-api] module to 70%
## Why you need it?
 Is your feature request related to a problem? Please describe in details

The test case coverage of [integration-tx-api] module is 18.33%, We need to go up to 70% to improve the robustness and stability of our code.
[integration-tx-api]模块的测试用例覆盖率为18.33%，我们需要提高到70%，以提高代码的健壮性和稳定性。

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

## Other related information
You can get some help from:
https://seata.apache.org/docs/developers/contributor-guide/test-coverage-guide_dev
https://seata.apache.org/blog/how-to-write-unit-tests

### PR DESCRIPTION
**Pull request by azatyamanaev**
- [x] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
increase test case coverage of [integration-tx-api] module

### Ⅱ. Does this pull request fix one issue?
fixes #6507

### CODE REVIEW

**Comment by xingfudeshi** [comment]
Pls register this PR in https://github.com/apache/incubator-seata/tree/2.x/changes

**Comment by azatyamanaev** [comment]
> Pls register this PR in https://github.com/apache/incubator-seata/tree/2.x/changes

Done

**Comment by azatyamanaev** [comment]
> Pls register this PR in https://github.com/apache/incubator-seata/tree/2.x/changes

@xingfudeshi Done

**Comment by azatyamanaev** [comment]
@xingfudeshi Can you resolve merge conflict?

**Comment by xingfudeshi** [comment]
> @xingfudeshi Can you resolve merge conflict?

Yes.The current version is being released.I'll resolve conflicts after the release.

**Comment by azatyamanaev** [comment]
> > @xingfudeshi Can you resolve merge conflict?
> 
> Yes.The current version is being released.I'll resolve conflicts after the release.

How long does it usually take?

**Comment by xingfudeshi** [comment]
> > > @xingfudeshi Can you resolve merge conflict?
> 
> > 
> 
> > Yes.The current version is being released.I'll resolve conflicts after the release.
> 
> 
> 
> How long does it usually take?


It won’t take long. After version 2.1 is released, I will start addressing the conflicting files in this PR.

File: changes/en-us/2.x.md [hunk 1]
```diff
@@ -153,6 +153,8 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6456](https://github.com/apache/incubator-seata/pull/6456)] adjust the test cases related to dynamic configuration
 - [[#6466](https://github.com/apache/incubator-seata/pull/6466)] support redis integration testing
 - [[#6484](https://github.com/apache/incubator-seata/pull/6484)] fix FileConfigurationTest and MockServerTest fail
+- [[#6545](https://github.com/apache/incubator-seata/pull/6545)] fix TestConfigCustomSPI compatibility test fail
+- [[#6533](https://github.com/apache/incubator-seata/pull/6533)] increase integration-tx-api module unit test coverage
```
Comment (lightClouds917) [inline changes/en-us/2.x.md:42]: alse need to add into zh-cn/2.x.md

**Review by lightClouds917 — CHANGES_REQUESTED** [review]

File: changes/en-us/2.x.md (hunk 1 above)
Comment (azatyamanaev) [reply changes/en-us/2.x.md:157]: Done

Comment (azatyamanaev) [reply changes/en-us/2.x.md:42]: > alse need to add into zh-cn/2.x.md

@lightClouds917 Done

**Review by lightClouds917 — APPROVED** [review]
please resolve the conflict

**Comment by azatyamanaev** [comment]
> please resolve the conflict

I can't, I don't have write access to repository. Or should I rebase my branch?

**Comment by lightClouds917** [comment]
> > please resolve the conflict
> 
> I can't, I don't have write access to repository. Or should I rebase my branch?
you can merge the 2.x branch to your branch,and resolve the confict.

**Comment by justinmclean** [comment]
I've fixed the conflicts for you.

**Review by lightClouds917 — APPROVED** [review]
LGTM

**Comment by funky-eyes** [comment]
我修复了codecov没有report成功的问题，现在让我们看看目前的单测覆盖度是多少
I've fixed the issue with Codecov not reporting successfully. Now, let's see what the current test coverage percentage is.

**Review by funky-eyes — APPROVED** [review]
LGTM

**Pull request merged**